package.json

```diff
@@ -16,11 +16,11 @@
   "license": "GPL-3.0",
   "devDependencies": {
     "@mate-academy/eslint-config": "*",
-    "@mate-academy/scripts": "^0.3.6",
+    "@mate-academy/scripts": "^1.2.8",
     "eslint": "^5.16.0",
     "eslint-plugin-jest": "^22.4.1",
     "eslint-plugin-node": "^8.0.1",
-    "jest": "^24.7.0"
+    "jest": "^29.5.0"
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use git add src/.